Gruntfile.js

```diff
@@ -355,19 +361,165 @@ module.exports = function (grunt) {
 							]
 						},
 						{
-							config: 'bump.version',
+							// if custom bump is used with a specific version, see dorelease task
+							config: 'release.buildSemVerType',
 							type: 'input',
 							message: 'What specific version would you like',
 							when: function (answers) {
-								return answers['bump.increment'] === 'custom';
+								return answers['release.buildSemVerType'] === 'custom';
 							},
 							validate: function (value) {
 								var valid = semver.valid(value);
 								return valid || 'Must be a valid semver, such as 1.2.3-rc1. See http://semver.org/ for more details.';
 							}
+						},
+						{
+							config: 'release.remoteRepository',
+							default : 'upstream',
+							type: 'input',
+							message: function() {
+								return 'What repository would like to base your local release branch from?';
+							}
+						},
+						{
+							// Assumption is made that you are releasing the code within a "release branch" currently 
+							// on the upstream remote repo. This branch will be tracked locally and be used to run 
+							// the build process in. It will be named release_{BUILD_VERSION}_{MMSS} (that is, it will
+							// use the version specified earlier and a "mini-timestamp" of the current hour and minute).
+							config: 'release.remoteBaseBranch',
+							type: 'input',
+							default : '3.x',
```

Comment:
The comment? Please elaborate. Are you asking for the two repo/branch prompts to merged and ask the person to write it with a slash?